lighthouse-core/audits/consistently-interactive.js

```diff
@@ -54,7 +54,7 @@ class ConsistentlyInteractiveMetric extends Audit {
    * @return {!Array<!TimePeriod>}
    */
   static _findNetworkQuietPeriods(networkRecords, traceOfTab) {
-    const traceEnd = traceOfTab.timestamps.traceEnd;
+    const traceEndTsInMs = traceOfTab.timestamps.traceEnd / 1000;
```

Comment:
aw I liked that each function in this file got this aliasing out of the way at the beginning :( 
did it make you look for its use too much right afterwards?